release.md

```diff
@@ -2,19 +2,132 @@
+    2. Otherwise follow instructions in [conda docs](https://conda.io/projects/conda/en/latest/user-guide/install/index.html)
+2. Install conda-smithy (conda-forge tool to update boilerplate in repo)
+    ```bash
+    conda install -n root -c conda-forge conda-smithy
+    ```
+3. Run conda-smithy on feedstock
+    ```bash
+    cd /path/to/feedstock/repo
+    conda-smithy rerender --commit auto
+    ```
+4. Push updated branch to the forked feedstock repo
+3. Make a PR on conda-forge/nlp_primitives-feedstock from the forked repo and let CI tests run - add "[DO NOT MERGE]" to the PR name to indicate this is PR should not be merged in
+4. After the tests pass, close the PR without merging
+
+## Create nlp_primitives release on Github
+
+#### Create release branch
+1. Branch off of master and name the branch the release version number (e.g. v0.4.0)
+
+#### Bump version number
 2. Bump verison number in `setup.py`, and `nlp_primitives/__init__.py`.
 
+#### Update changelog
+1. Replace "Future Release" in `docs/source/changelog.rst` with the current date
+    ```
+    **v0.4.0** Aug 11, 2020
+    ```
+2. Remove any unused changelog sections for this release (e.g. Fixes, Testing Changes)
+3. Add yourself to the list of contributors to this release and put the contributors in alphabetical order
+4. The release PR does not need to be mentioned in the list of changes
+
 ## Create Release PR
-A release PR should have the version number as the title and the changelog updates as the PR body text. The contributors line is not necessary.
+A release PR should have the version number as the title and the changelog updates as the PR body text. The contributors line is not necessary. The special sphinx docs syntax (:pr:`547`) needs to be changed to github link syntax (#547).
 
 ## Create GitHub Release
 After the release pull request has been merged into the master branch, it is time draft the github release.
 * The target should be the master branch
-* The tag should be the version number with a v prefix (e.g. v0.1.2)
+* The tag should be the version number with a v prefix (e.g. v0.4.0)
 * Release title is the same as the tag
-* Release description should be the full changelog updates for the release, including the line thanking contributors.
+* Release description should be the full changelog updates for the release, including the line thanking contributors. Contributors should also have their links changed from the docs syntax (:user:`rwedge`) to github syntax (@rwedge)
+* This is not a pre-release
+* Publishing the release will automatically upload the package to PyPI
 
-## Release on PyPI
-After publishing the release on GitHub, a GitHub Action will automatically upload the package to PyPI.
\ No newline at end of file
+## Release on conda-forge
+1. A bot should automatically create a new PR in conda-forge/nlp_primitives-feedstock
+2. Update requirements changes in `recipe/meta.yaml` (bot should have handled version and source links on its own)
```

Comment:
Non-maintainers will need a maintainer to merge their changes into the bot's pr

Reply:
Added